pkgs/applications/audio/netease-music-tui/default.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchFromGitHub, rustPlatform, lib, alsaLib, pkg-config, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  name = "netease-music-tui-${version}";
+  version = "v0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "betta-cyber";
+    repo = "netease-music-tui";
+    rev = "${version}";
+    sha256 = "0m5b3q493d32kxznm4apn56216l07b1c49km236i03mpfvdw7m1f";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
Did you submit this as a PR? If so fetch that patch.

Reply:
The pr in upstream is on readme file about prerequisites. The patch that used in this pr is on cargo.lock, which is similar to other packages by rust without a lock file in original repo.

Reply:
Can you submit it so that we can fetch it?

Reply:
The author ignore that file from repo. And take a look like [othter rust package on nixpkgs](https://github.com/NixOS/nixpkgs/blob/2c931312ce6c7baf91ec98e164bef0241fc20aa6/pkgs/applications/misc/effitask/default.nix), I do the same thing. Why must we fetch it from different place.

Reply:
I wanted to upstream the change to not necessarily add the large patch to the git but it is no required.